### PR TITLE
fix inconsistencies between docs and deployment

### PR DESCRIPTION
**Pull request by willharrison**

### CODE REVIEW

File: _api/reference/lock.md [hunk 1]
```diff
@@ -16,17 +16,19 @@ Represents a Lock on the UniKey CORE Platform, such as a User's Kwikset Kevo.  L
 | - | - |
 | `id` | UUID | The ID of the Lock |
 | `name` | String | The name of the Lock created by the User |
-| `bolt_state` | String | The current Bolt State (if available) |
+| `bolt_status` | String | The current Bolt State (if available) |
```
Comment (jlorich) [inline _api/reference/lock.md]: State is correct for what we have been using.

File: _api/reference/lock.md [hunk 2]
```diff
@@ -41,15 +43,15 @@ Represents a Lock on the UniKey CORE Platform, such as a User's Kwikset Kevo.  L
 ## Available APIs
 
 
-##### `GET` `/locks/{lock_id}`
+##### `GET` `/locks/{LockId}`
```
Comment (jlorich) [inline _api/reference/lock.md]: no pascal case, use underscore

File: _api/reference/lock.md [hunk 3]
```diff
@@ -41,15 +43,15 @@ Represents a Lock on the UniKey CORE Platform, such as a User's Kwikset Kevo.  L
...
 
 Returns information about a given Lock
 
 ###### Request Properties
 
 | Name | Type | Description |
 | - | - |
-| `lock_id` | UUID | **Required** The ID of the desired Lock |
+| `LockId` | UUID | **Required** The ID of the desired Lock |
```
Comment (jlorich) [inline _api/reference/lock.md]: no pascal case

File: _api/reference/oauth.md [hunk 4]
```diff
@@ -34,7 +34,8 @@ Exchange an authorization code (returned from the [Authentication Process](/api/
 | `grant_type` | String | **Required** The type of token we'll be granting (`authorization_code` or `refresh_token`) |
 | `client_id` | UUID | **Required** The *application* ID provided to you by UniKey |
 | `client_secret` | String | **Required** The *application* secret provided to you by UniKey |
-| `code` | String | **Required** The `authorization_code` you received from Step 1  |
+| `code` | String | The `authorization_code` you received from Step 1  |
```
Comment (jlorich) [inline _api/reference/oauth.md]: authorization code should be required right?

Comment (willderness) [inline _api/reference/oauth.md]: I thought you said this was auto generated? Where is it generated from?

**Comment by Jakkob** [comment]
The markdown is handwritten. We use Jekyll to generate the static HTML.

**Comment by willderness** [comment]
Oh word, I was thinking that the docs were generated out of some javadoc or
something similar. Cool

**Comment by Jakkob** [comment]
I'd love to switch the Public API to Swagger at some point.

File: _api/reference/lock.md (hunk 1 above)
Comment (willharrison) [inline _api/reference/lock.md]: status is what the current API returns

**Pull request merged**